scripts/lib/mkcloud-driver-libvirt.sh

```diff
@@ -51,7 +51,7 @@ function libvirt_start_daemon()
 
 function libvirt_net_start()
 {
-    $sudo virsh net-start $cloud-admin
+    $sudo virsh net-start $cloud-admin || exit $?
```

Comment:
Does virsh print a useful error message? Otherwise we could use our "complain" function here.

Reply:
It does, from my testing with ipv6 patches:

```
defining network from vp4-admin
defining network from /tmp/vp4-admin.net.xml
error: Failed to start network vp4-admin
error: internal error: Check the host setup: enabling IPv6 forwarding with RA routes without accept_ra set to 2 is likely to cause routes loss. Interfaces to look at: bond0

mkchp:~/manual.stevenk # 
```